fenix/app/src/main/res/values/strings.xml

```diff
@@ -1596,7 +1596,10 @@
     back from ETP details (Ex: Tracking content) -->
     <string name="etp_back_button_content_description">Navigate back</string>
     <!-- About page link text to open what's new link -->
-    <string name="about_whats_new">What’s new in %s</string>
+    <string name="about_whats_new" moz:RemovedIn="122" tools:ignore="UnusedResources">What’s new in %s</string>
+    <!-- About page link text to open the latest release notes link. Fenix only has release notes for 'Release' versions, NOT 'Beta' & 'Nightly'.
+         Note: The word "Firefox" should NOT be translated -->
+    <string name="about_whats_new_2">What’s new in Firefox</string>
```

Comment:
Is there a reason we can't use another placeholder here, like we usually do with app names?
So the placeholder would always be replaced by "Firefox" (with no flavor to it)
We would have to call out what the placeholder will be replaced with at run time in the comment (it's currently missing, must have fallen through the cracks at the time the string landed)

Reply:
Example: https://transvision.mozfr.org/?recherche=%251%24s+is+produced+by+Mozilla.&repo=android_l10n&sourcelocale=en-US&locale=fr&search_type=strings

Reply:
That's a great suggestion @Delphine 🙂